test/no-ripple-test.js

```diff
@@ -1,15 +1,14 @@
-var async       = require('async');
-var assert      = require('assert');
-var ripple      = require('ripple-lib');
-var Amount      = require('ripple-lib').Amount;
-var Remote      = require('ripple-lib').Remote;
-var Transaction = require('ripple-lib').Transaction;
-var Server      = require('./server').Server;
-var testutils   = require('./testutils');
-var config      = testutils.init_config();
+let async       = require('async');
+let assert      = require('assert-diff');
+let ripple      = require('ripple-lib');
+let Amount      = require('ripple-lib').Amount;
+let Remote      = require('ripple-lib').Remote;
+let testutils   = require('./testutils');
+
+let {alice,bob,carol,dan,root,mtgox,bitstamp} = require('./testconfig').accounts;
```

Comment:
One of the reason the tests which rely upon coffee-script fails is probably to do with the fact that it monkey patched the config to add aliases on the fly, in much the same way as these accounts were added.